Project.toml

```diff
@@ -81,19 +81,19 @@ RecipesBase = "1.1"
 Reexport = "1.0"
 Requires = "1.1"
 SciMLBase = "1.90, 2"
-Setfield = "0.8, 1"
+Setfield = "1"
 SimpleUnPack = "1.1"
 SparseArrays = "1"
-StartUpDG = "0.17"
-Static = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8"
+StartUpDG = "0.17.7"
+Static = "0.8.7"
 StaticArrayInterface = "1.4"
-StaticArrays = "1"
-StrideArrays = "0.1.18"
-StructArrays = "0.6"
+StaticArrays = "1.5"
+StrideArrays = "0.1.26"
+StructArrays = "0.6.11"
 SummationByPartsOperators = "0.5.41"
-T8code = "0.4.3, 0.5"
+T8code = "0.4.3"
```

Comment:
This seems like a mistake or do we not support the current T8code?

Reply:
Yes, similar to above.